hack/generate/samples/internal/ansible/molecule.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
@fabianvf and @asmacdo 
Now, the Molecule mock data is gen and we update it as we need to. So, is no longer required the dirs 
```
├── test
│   ├── ansible
│   │   ├── OWNERS
│   │   ├── ansible.cfg
│   │   ├── build
│   │   │   └── Dockerfile
│   │   ├── deploy
│   │   │   ├── crds
│   │   │   │   ├── test.example.com_argstest_crd.yaml
│   │   │   │   ├── test.example.com_casetest_crd.yaml
│   │   │   │   ├── test.example.com_collectiontests_crd.yaml
│   │   │   │   ├── test.example.com_inventorytests_crd.yaml
│   │   │   │   ├── test.example.com_reconciliationtest_crd.yaml
│   │   │   │   ├── test.example.com_selectortest_crd.yaml
│   │   │   │   ├── test.example.com_subresourcestests_crd.yaml
│   │   │   │   ├── test.example.com_v1_inventorytest_cr.yaml
│   │   │   │   └── test.example.com_v1alpha1_inventorytest_cr.yaml
│   │   │   ├── operator.yaml
│   │   │   ├── role.yaml
│   │   │   ├── role_binding.yaml
│   │   │   └── service_account.yaml
│   │   ├── fixture_collection
│   │   │   ├── galaxy.yml
│   │   │   └── roles
│   │   │       └── dummy
│   │   │           └── tasks
│   │   │               └── main.yml
│   │   ├── inventory
│   │   │   ├── group_vars
│   │   │   │   └── test.yml
│   │   │   └── hosts
│   │   ├── molecule
│   │   │   ├── cluster
│   │   │   │   ├── converge.yml
│   │   │   │   ├── create.yml
│   │   │   │   ├── destroy.yml
│   │   │   │   ├── molecule.yml
│   │   │   │   ├── prepare.yml
│   │   │   │   ├── tasks
│   │   │   │   │   ├── args_test.yml
│   │   │   │   │   ├── case_test.yml
│   │   │   │   │   ├── collections_test.yml
│   │   │   │   │   ├── inventory_test.yml
│   │   │   │   │   ├── liveness_test.yml
│   │   │   │   │   ├── reconciliation_test.yml
│   │   │   │   │   ├── secrets_test.yml
│   │   │   │   │   ├── selector_test.yml
│   │   │   │   │   └── subresources_test.yml
│   │   │   │   └── verify.yml
│   │   │   ├── default
│   │   │   │   └── prepare.yml
│   │   │   ├── templates
│   │   │   │   └── operator.yaml.j2
│   │   │   └── test-local
│   │   │       ├── converge.yml
│   │   │       ├── molecule.yml
│   │   │       ├── prepare.yml
│   │   │       └── verify.yml
│   │   ├── playbooks
│   │   │   ├── args.yml
│   │   │   ├── case.yml
│   │   │   ├── inventory.yml
│   │   │   ├── reconciliation.yml
│   │   │   ├── secret.yml
│   │   │   ├── selector.yml
│   │   │   └── subresources.yml
│   │   ├── plugins
│   │   │   └── filter
│   │   │       └── test.py
│   │   ├── requirements.yml
│   │   ├── roles
│   │   │   └── inventory
│   │   │       └── tasks
│   │   │           └── main.yml
│   │   └── watches.yaml
 
```
And copy the full directories which not allow are we not let we testing all changes with the latest changes in the scaffold since it means that we should ensure that these mock is also updated/generated based on the PR. See;  https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-ansible-molecule.sh#L34-L50